design/workflow.go

```diff
@@ -114,6 +114,15 @@ var _ = Service("workflow", func() {
 
 		HTTP(func() {
 			POST("/workflows/assign")
+			Param("workflowName", String, "Name of the Workflow to be associated with the codeset", func() {
```

Comment:
Wait, what about having just `name` here? The command is already `workflow assign`, so it could be implied that simple `name` belongs to workflow...

Reply:
I agree with @jsuchome . In addition to that, you don't need to duplicate the information you already have in the payload section, you can just have something like:
```
Param("workflowName")
```
The catch is you need to use the same name as in the payload section and, for some reason unknown to me, you need a payload field for every parameter (i.e. you can't only have parameters).

Reply:
nice, I have updated it with suggested changes.